logger/winston.logger.ts

```diff
@@ -0,0 +1,33 @@
+import { createLogger, format, transports } from 'winston';
+
+const customFormat = format.printf(({ timestamp, level, stack, message }) => {
+  return `${timestamp} - [${level.toUpperCase().padEnd(1)}] - ${stack || message}`;
+});
+
+const options = {
+  file: {
+    filename: './logger/error.log',
+    level: 'error',
+  },
+  console: {
+    level: 'silly',
+  },
+};
+
+const devLogger = {
+  format: format.combine(
+    format.timestamp(),
+    format.errors({ stack: true }),
+    customFormat,
+  ),
+  transports: [
+    new transports.File(options.file),
+    new transports.File({
+      filename: './logger/combine.log',
+      level: 'info',
+    }),
+    new transports.Console(options.console),
+  ],
+};
+
+export const instance = createLogger(devLogger);
```

Comment:
jak to logger któego mozna używać to raczej powinien się nazywać WinstonLogger w exporcie